css/index.css

```diff
@@ -143,4 +143,80 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.head{
+    display: flex;
+    justify-content: space-around;
+}
+
+.head nav{
+    display: flex;
+    justify-content: space-around;
+    flex-basis: 80%;
+}
+nav a {
+    padding-top: 20px;
+    color: black;
+}
+
+nav a:hover{
+    background-color: white;
+    color: red;
+
+}
+
+.jumbo{
+    margin-top: 30px;
+    margin-bottom: 20px;
+}
+
+.section1{
+    display: flex;
+    margin: 0 auto;
+    justify-content: space-between;
+    margin: 50px 0px 30px 0px;
+
+    
+}
+
+.section2{
+    display: flex;
+    margin: 0 auto;
+    justify-content: space-between;
+}
+
+.about{
+    margin: 15px 0px 25px 0px;
+}
+.jumbo-about{
+    margin:20px 0px 20px 0px;
+}
+.strategy{
+    flex-wrap: wrap;
+}
+.work, .places{
+    flex-wrap: wrap;
+}
```

Comment:
Since the above three classes, strategy, work and places all have the same style, you could instead use one generic class name such as `infobox`.